src/python_testing/TC_EWATERHTR_2_2.py

```diff
@@ -259,6 +294,8 @@ async def test_TC_EWATERHTR_2_2(self):
         self.step("11")
         await self.send_test_event_trigger_water_temperature61C_test_event()
 
+        event_data = events_callback.wait_for_event_report(Clusters.WaterHeaterManagement.Events.BoostEnded)
```

Comment:
As above - do we need an exception / assert here?

Reply:
See above